spec/lib/promises.helper.spec.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+describe("promises.helper", () => {
+    var check, mockRequire;
+
+    mockRequire = require("mock-require");
+
+    /**
+     * Provides a way of easily passing in a mock Jasmine environment.
+     *
+     * @param {string} filePath
+     * @param {number} version
+     * @param {Object} optionalMock Only passed in to test that patching
```

Comment:
At the expense of clearer documentation though? Maybe the comment isn't necessary at all?